ihmc-footstep-planning/src/main/resources/us/ihmc/footstepPlanning/SnappingTerrainExtractor.cu

```diff
@@ -246,28 +237,19 @@ __global__ void computeTerrainData(float *heightMap, size_t pitchHeightMap,
                 // compute the relative height at this point, compared to the height contained in the current cell.
                 float relative_height_of_query = query_height_float - snap_height;
```

Comment:
Instead of using the height at the middle of the "foot"/circular footprint, the best-fit plane height could be used. That's how it's done in HeightMapCliffAvoider.